scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -375,10 +378,17 @@ main (int argc, char *argv[])
                    "Ssid", SsidValue (ssid),
                    "BE_MaxAmpduSize", UintegerValue (mpduAggregationSize),
                    "BE_MaxAmsduSize", UintegerValue (msduAggregationSize),
+                   "BK_MaxAmpduSize", UintegerValue (mpduAggregationSize),
+                   "BK_MaxAmsduSize", UintegerValue (msduAggregationSize),
+                   "VI_MaxAmpduSize", UintegerValue (mpduAggregationSize),
+                   "VI_MaxAmsduSize", UintegerValue (msduAggregationSize),
                    "SSSlotsPerABFT", UintegerValue (8), "SSFramesPerSlot", UintegerValue (13),
                    "BeaconInterval", TimeValue (MicroSeconds (102400)),
                    "ATIPresent", BooleanValue (false));
 
+  wifiMac.SetAttribute ("VO_MaxAmpduSize", UintegerValue (mpduAggregationSize),
+                        "VO_MaxAmsduSize", UintegerValue (msduAggregationSize));
+
```

Comment:
I'd put all max packet size definitions either in SetType or SetAttrubute, but it's just for aesthetics